src/main/java/com/backbase/ct/bbfuel/setup/AccessControlSetup.java

```diff
@@ -254,6 +259,13 @@ private void ingestTransactions(List<ArrangementId> arrangementIds, boolean isRe
         }
     }
 
+    private void ingestPockets(List<ArrangementId> arrangementIds, boolean isRetail) {
+        if (this.globalProperties.getBoolean(PROPERTY_INGEST_POCKETS)) {
+            arrangementIds.parallelStream()
+                .forEach(arrangementId -> this.pocketsConfigurator.ingestPockets(arrangementId, isRetail));
```

Comment:
I think we should ingest pockets for only one of the user's arrangements, we should create the parent arrangement and then ingest pockets for it.

Reply:
How do I create the parent arrangement? And that parent arrangement should ingest the pockets?

Reply:
Check how this method is used: https://github.com/Backbase/bb-fuel/blob/master/src/main/java/com/backbase/ct/bbfuel/client/productsummary/ArrangementsIntegrationRestClient.java#L37

Reply:
I need some more instructions to be able to create a parent arrangement. Should it be in the ingestPocket method or before it is called? What I use as input data?

Reply:
You need to make this call work: https://stash.backbase.com/projects/PFM/repos/pocket-management/browse/pocket-tailor/src/main/java/com/backbase/dbs/pfm/pockets/service/impl/PocketTailorServiceImpl.java#62

There is a mock service that always returns the same arrangement when it's asked to created a parent one: https://stash.backbase.com/projects/SDLC/repos/beck-helm/browse/helmfile.yaml#2227

Reply:
Attempt to implement pocket parent arrangement, but config of PocketsMockArrangementRestClient results in ERROR: `nodename nor servname provided, or not known`
I do not know how to solve this @lehphyro .

Reply:
That means the service you're trying to call doesn't exist.

Reply:
PocketsMockArrangementRestClient , you are trying to use arrangement outbound mock service which is deployed as follows 
https://stash.backbase.com/projects/SDLC/repos/beck-helm/browse/helmfile.yaml#2221. please change pocketsArrangements: http://app.${environment.name}.${environment.domain}/pocketsprovidermock/service-api
 
in src/main/resources/application-k8s-beta.yml

Reply:
I used the host config value from helm file...